superset/views/core.py

```diff
@@ -298,6 +299,65 @@ class DatabaseAsync(DatabaseView):
 appbuilder.add_view_no_menu(DatabaseAsync)
 
 
+class CsvToDatabaseView(SimpleFormView):
+    form = CsvToDatabaseForm
+    form_title = _('CSV to Database configuration')
+    add_columns = ['database', 'schema', 'table_name']
+
+    def form_get(self, form):
+        form.sep.data = ','
```

Comment:
I think the normal way of setting defaults for wtforms is to do it when creating the form objects. As in:
`BooleanField('name', default=True, ...)`